src/dreammaker/lexer.rs

```diff
@@ -77,6 +77,7 @@ table! {
     "%=",  ModAssign;
     "&",   BitAnd;
     "&&",  And;
+    "&&=", SafeBitAndAssign;
```

Comment:
Wouldn't `AndAssign` be a more accurate name? If `x &&= y` is like `x = x && y`, then it's not "safe" and it's not "bit". Ditto `||=`

Reply:
Yeah. That would make a lot more sense. Was thinking of how ?[] works for some reason when I named these.